src/Components/Facility/TriageForm.tsx

```diff
@@ -263,54 +251,29 @@ export const TriageForm = (props: triageFormProps) => {
           },
         }}
         backUrl={`/facility/${facilityId}`}
-      />
-
-      <Modal
-        open={openModalForExistingTriage}
-        aria-labelledby="Triage Check"
-        aria-describedby=""
-        className=""
       >
-        <div className="h-screen w-full absolute flex items-center justify-center bg-modal">
-          <div className="bg-white rounded shadow p-8 m-4 max-h-full text-center flex flex-col max-w-lg w-2/3 min-w-max-content">
-            <div className="mb-4">
-              <i className="fa-solid fa-triangle-exclamation text-red-500 fa-4x"></i>
-              <h1 className="sm:text-xl text-sm">
+        <ConfirmDialogV2
+          title={
+            <div className="">
+              <i className="fa-solid fa-triangle-exclamation text-red-500 fa-3x"></i>
+              <h1 className="sm:text-xl text-sm mt-4">
```

Comment:
Avoid setting your own sizes for the title. The confirm dialog does that for you for texts passed in the title. This is to keep designs consistent and easy to maintain.

Reply:
Done. used a <p> tag for the text without any css